src/components/Navigation.js

```diff
@@ -0,0 +1,27 @@
+import { Link } from 'react-router-dom';
+import avatar from './assets/avatar.png';
+
+function Navigation() {
+  return (
+    <nav className="nav">
+      <ul className="nav-items">
+        <li className="nav-link">
+          <Link to="/" className="link">
+            <h1 className="logo"> BookStor CMS </h1>
+          </Link>
+        </li>
+        <li className="nav-link">
+          <Link to="/" className="link"> Books </Link>
+        </li>
+        <li className="nav-link">
+          <Link to="/categories" className="link"> Categories </Link>
+        </li>
```

Comment:
Your navigation list looks good but please make sure to use the same color and add the active effect to match the template 

![Capture d’écran du 2023-08-04 22-17-16](https://github.com/akbarsazish/Bookstore-React/assets/59707859/49dcb383-1490-4c67-883d-6390b511fc11)
![Capture d’écran du 2023-08-04 22-17-07](https://github.com/akbarsazish/Bookstore-React/assets/59707859/0719cf7d-e928-43aa-962e-655f6934c168)